Kinder/Classes/FileManager.cs

```diff
@@ -33,12 +36,19 @@ class FileManager
                     users.Add(user);
                 }
             }
+
             return users;
         }
-
-        public void saveUser()
+        public static void addNewUser(string Username, string Password, string Email, string PhoneNumber, string Name, string Surname, int id)
         {
-            //TODO
+            using (StreamWriter sw = new StreamWriter(FileLocation, true))
+            {
+                sw.Write(id);
+                sw.Write(' ');
+                sw.WriteLine(Username + ' ' + Password + ' ' + Email + ' ' + PhoneNumber + ' ' + Name + ' ' + Surname + ' ' + 0);
```

Comment:
Think this can be fixed by making id a string using id.ToString() and then putting everything in one line.